<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignupWindow</class>
 <widget class="QMainWindow" name="SignupWindow">
  <property name="geometry">
   <rect><x>0</x><y>0</y><width>800</width><height>600</height></rect>
  </property>
  <property name="windowTitle">
   <string>Sign up</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="rootLayout">
    <property name="spacing"><number>16</number></property>

    <item>
     <spacer name="topSpacer">
      <property name="orientation"><enum>Qt::Vertical</enum></property>
      <property name="sizeHint"><size><width>20</width><height>50</height></size></property>
     </spacer>
    </item>

    <item>
     <layout class="QHBoxLayout" name="centerRow">
      <item>
       <spacer name="leftSpacer">
        <property name="orientation"><enum>Qt::Horizontal</enum></property>
        <property name="sizeHint"><size><width>100</width><height>20</height></size></property>
       </spacer>
      </item>

      <item>
       <widget class="QFrame" name="card">
        <property name="minimumSize"><size><width>420</width><height>0</height></size></property>
        <property name="maximumSize"><size><width>480</width><height>16777215</height></size></property>
        <property name="frameShape"><enum>QFrame::StyledPanel</enum></property>
        <layout class="QVBoxLayout" name="cardLayout">
         <property name="spacing"><number>12</number></property>

         <item>
          <widget class="QLabel" name="titleLabel">
           <property name="text"><string>Create account âœ¨</string></property>
           <property name="alignment"><set>Qt::AlignCenter</set></property>
          </widget>
         </item>

         <item>
          <widget class="QLineEdit" name="editName">
           <property name="placeholderText"><string>Name</string></property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editUsername">
           <property name="placeholderText"><string>Username</string></property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editPhone">
           <property name="placeholderText"><string>Phone Number (09xxxxxxxxx)</string></property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editEmail">
           <property name="placeholderText"><string>Email</string></property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editPassword">
           <property name="echoMode"><enum>QLineEdit::Password</enum></property>
           <property name="placeholderText"><string>Password (min 8 chars)</string></property>
          </widget>
         </item>

         <item>
          <widget class="QPushButton" name="btnSignup">
           <property name="text"><string>Sign up</string></property>
          </widget>
         </item>

         <item>
          <widget class="QPushButton" name="btnBack">
           <property name="text"><string>Back</string></property>
          </widget>
         </item>

        </layout>
       </widget>
      </item>

      <item>
       <spacer name="rightSpacer">
        <property name="orientation"><enum>Qt::Horizontal</enum></property>
        <property name="sizeHint"><size><width>100</width><height>20</height></size></property>
       </spacer>
      </item>
     </layout>
    </item>

    <item>
     <spacer name="bottomSpacer">
      <property name="orientation"><enum>Qt::Vertical</enum></property>
      <property name="sizeHint"><size><width>20</width><height>50</height></size></property>
     </spacer>
    </item>

   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
